pkg/pipeline/encode/encode_prom.go

```diff
@@ -32,7 +32,7 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-const defaultExpiryTime = 2 * time.Minute
+const defaultExpiryTime = time.Duration(2 * time.Second)
```

Comment:
Was the change of the default from 2 minutes to 2 seconds intentionally?

Reply:
corrected